docs/source/guide/dataset_create.md

```diff
@@ -20,26 +20,217 @@ hide_sidebar: true
+
+    Leave **Provide user access to the AWS Management Console** unselected. Click **Next**.
+
+5. Select **Attach policies directly**.
+
+6. Under **Permissions policies**, use the search field to find and select the policy you are using with the user (see above). Click **Next**.
+
+7. Click **Create user**.
+
+
+
+###### Generate an access key for the user
+
+1. From the **Users** page, click the user you created in the previous section.  
+
+2. Click the **Security Credentials** tab.
+
+    ![Screenshot of the Security Credentials option](/images/data_discovery/aws_key.png)
+
+3. Scroll down to **Access keys** and click **Create access key**.
+4. Select **Other** and note the recommendations provided by AWS. Click **Next**.
+5. Optionally, add a description for the key.
+6. Click **Create access key**.
+7. Copy the access key ID and your secret access key and keep them somewhere safe, or export the key to a CSV file.
+
+    ![Screenshot of the copy icon next to the key](/images/data_discovery/aws_key2.png)
+
+    <div class="admonition attention"><p class="admonition-title">Important</p><p>This is the only time you will be able to copy the secret access key. Once you click <strong>Done</strong>, you will not be able to view or copy it again.</p></div>
+
+8. Click **Done**. 
+   
+{% enddetails %}
+
+### Create a dataset from an AWS S3 bucket
+
+1. From Label Studio, navigate to the Datasets page and click **Create Dataset**. 
+
+    ![Create a dataset action](/images/data_discovery/dataset_create.png)
+
+2. Complete the following fields and then click **Next**:
+
+    <div class="noheader rowheader">
+
+    | | |
+    | --- | --- |
+    | Name | Enter a name for the dataset. |
+    | Description | Enter a brief description for the dataset.  |
+    | Source | Select AWS S3. |
+
+    </div>
+
+3. Complete the following fields: 
+
+    <div class="noheader rowheader">
+
+    | | |
+    | --- | --- |
+    | Bucket Name | Enter the name of the AWS S3 bucket. |
+    | Bucket Prefix | Enter the folder name within the bucket that you would like to use.  For example, `data-set-1` or `data-set-1/subfolder-2`.  |
+    | File Name Filter | Use glob format to filter which file types to sync. For example, to sync all JPG files, enter `*jpg`. To sync all JPG and PNG files, enter `**/**+(jpg\|png)`.<br><br>At this time, we support the following file types: .jpg, .jpeg, .png, .txt, .text |
+    | Region Name | By default, the region is `us-east-1`. If your bucket is located in a different region, enter it here.  |
+    | S3 Endpoint | Enter an S3 endpoint if you want to override the URL created by S3 to access your bucket. |
+    | Access Key ID | Enter the ID for the access key you created in AWS. Ensure this access key has read permissions for the S3 bucket you are targeting (see "Create an AWS access key" above). |
+    | Secret Access Key | Enter the secret portion of the access key you created earlier. |
```

Comment:
I know we discussed this, so apologies if I'm forgetting a discussion point, but if there's an obvious page to link to in the AWS docs for this I think it'd be helpful. You and I were both confused about this, so I'm guessing we'll help some users out if we can link to this for them.